### ISSUE TITLE
Update Russian translations

### ISSUE THREAD

**Issue by bndby** [squidfunk/mkdocs-material#4962]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "ru",
  "action.edit": "Редактировать страницу",
  "action.skip": "Перейти к содержанию",
  "action.view": "Посмотреть исходный код страницы",
  "announce.dismiss": "Больше не показывать",
  "blog.archive": "Архив",
  "blog.categories": "Категории",
  "blog.categories.in": "В",
  "blog.continue": "Читать",
  "blog.draft": "Черновик",
  "blog.index": "На главную",
  "blog.meta": "Метаданные",
  "blog.references": "Ссылки",
  "clipboard.copy": "Копировать в буфер",
  "clipboard.copied": "Скопировано в буфер",
  "consent.accept": "Принять",
  "consent.manage": "Управлять настройками",
  "consent.reject": "Отклонить",
  "footer": "Нижний колонтитул",
  "footer.next": "Вперед",
  "footer.previous": "Назад",
  "header": "Верхний колонтитул",
  "meta.comments": "Комментарии",
  "meta.source": "Исходный код",
  "nav": "Навигация",
  "readtime.one": "Читать 1 минуту",
  "readtime.other": "Читать # минут",
  "rss.created": "RSS канал",
  "rss.updated": "RSS канал с новым контентом",
  "search": "Поиск",
  "search.placeholder": "Поиск",
  "search.share": "Поделиться",
  "search.reset": "Очистить",
  "search.result.initializer": "Инициализация поиска",
  "search.result.placeholder": "Начните печатать для поиска",
  "search.result.none": "Совпадений не найдено",
  "search.result.one": "Найдено 1 совпадение",
  "search.result.other": "Найдено совпадений: #",
  "search.result.more.one": "Ещё 1 на этой странице",
  "search.result.more.other": "Ещё # на этой странице",
  "search.result.term.missing": "Отсутствует",
  "select.language": "Выберите язык",
  "select.version": "Выберите версию",
  "source": "Перейти к репозиторию",
  "source.file.contributors": "Участники",
  "source.file.date.created": "Дата создания",
  "source.file.date.updated": "Последнее обновление",
  "tabs": "Вкладки",
  "toc": "Содержание раздела",
  "top": "К началу"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in 607c31661 😊

**Comment by squidfunk** [comment]
Released as part of [9.0.10](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.10).
